docs/pages/cloud/downloads.mdx

```diff
@@ -51,6 +51,33 @@ Linux Debian Packages              | [amd64](https://get.gravitational.com/telep
 Linux RPM Packages                 | [amd64](https://get.gravitational.com/teleport-ent-(=cloud.version=)-1.x86_64.rpm) | [arm64](https://get.gravitational.com/teleport-ent-(=cloud.version=)-1.arm64.rpm) | [armv7](https://get.gravitational.com/teleport-ent-(=cloud.version=)-1.arm.rpm) | [i386](https://get.gravitational.com/teleport-ent-(=cloud.version=)-1.i386.rpm) |
 macOS Packages                     | [amd64](https://get.gravitational.com/teleport-ent-(=cloud.version=).pkg) |
 
+## Teleport package repositories
+
+Teleport currently maintains DEB repositories for Debian compatible systems.
+
+<Tabs>
```

Comment:
I think we should convert the `Tabs` component to regular body text since there is only one `TabItem`.

Reply:
This is on purpose, there will be a 2nd tab soon when we roll out RPM repos.

Reply:
Added a tab with RPM repos too now.